assets/blocks/course-progress-block/course-progress-edit.js

```diff
@@ -76,18 +76,17 @@ export const CourseProgressEdit = ( props ) => {
 			borderRadius,
 		},
 	};
-
+	const progressBarLabel = _n( 'lesson', 'lessons', 2, 'sensei-lms' );
 	return (
 		<>
 			<ProgressBar
-				lessonsCount={ totalLessonsCount }
+				totalCount={ totalLessonsCount }
 				completedCount={ completedLessonsCount }
 				wrapperAttributes={ wrapperAttributes }
 				barWrapperAttributes={ barWrapperAttributes }
 				barAttributes={ barAttributes }
 				countersClassName="wp-block-sensei-lms-progress-heading"
-				lessonsCountClassName="wp-block-sensei-lms-progress-heading__lessons"
```

Comment:
Lesson one is not used anymore since we don't have a different styling for different parts of text, we only use second part